src/pages/Home/HomeMultiversX.tsx

```diff
@@ -68,14 +71,27 @@ export default function HomeMultiversX({
 
   const navigate = useNavigate();
 
-  const mxFaucetContract = new FaucetContract(_chainMeta.networkId);
-  const mxClaimsContract = new ClaimsContract(_chainMeta.networkId);
+  useEffect(() => {
```

Comment:
This is redundant beacuse AppHarness only renders its children when chainMeta is set